google-cloud-debugger/lib/google/cloud/debugger/credentials.rb

```diff
@@ -38,9 +36,14 @@ module Debugger
       #
       #   debugger.project_id #=> "my-project"
       #
-      class Credentials < Google::Cloud::Debugger::V2::Credentials
-        SCOPE = (Google::Cloud::Debugger::V2::Credentials::SCOPE +
-                 Google::Cloud::Logging::Credentials::SCOPE).uniq.freeze
+      class Credentials < Google::Cloud::Debugger::V2::Debugger::Credentials
+        self.scope = [
+          "https://www.googleapis.com/auth/cloud-platform",
+          "https://www.googleapis.com/auth/cloud_debugger",
+          "https://www.googleapis.com/auth/logging.admin",
+          "https://www.googleapis.com/auth/logging.read",
+          "https://www.googleapis.com/auth/logging.write"
+        ]
```

Comment:
This scopes array also appears in `google-cloud-debugger.rb`, above. Would it be possible to DRY this up somehow?

Reply:
Yeah, there are actually a number of values that are duplicated between this config and other places (credentials classes, generated configs and clients, etc.) We need to do a general cleanup and reconciliation of the veneer configs, but that's going to require some careful design work, and unless these are actually broken I don't want to rush to change things until we've done that design work.